### DISCUSSION TITLE
How to Use with an ExpressJs App

### DISCUSSION THREAD

**Discussion by thecodeclimber** [parcel-bundler/parcel#4945] [General]
I would like to use parcel with my express.js app.  However, I can't figure how to integrate the two together.  Are there any tutorials or any suggestions on how to get parcel to work with an express app?

Thanks.

**Comment by DeMoorJasper** [comment]
What exactly do you want to do, if you want to bundle your application using parcel it is just bundling nodejs.
However if you want to run parcel on the fly I'd advice against it unless you're creating a custom devserver.

**Comment by thecodeclimber** [comment]
I want to bundle my assets (css, js, images, etc) into a public folder that I then run via express.
